### PR TITLE
fix: remove .gz extension for containerize, trim tests

### PR DESCRIPTION
**Pull request by tomberek**
## Proposed Changes
- removes ".gz" extension when producing a ".tar"
- remove superfluous tests that take a long time (151s -> 41s)

## Release Notes
Fixes: https://github.com/flox/product/issues/769

### CODE REVIEW

**Review by stahnma — APPROVED** [review]

File: cli/tests/containerize.bats [hunk 1]
```diff
@@ -131,17 +113,18 @@ function skip_if_linux() {
...
 
-  run podman load -i test-container.tar.gz
+  run podman load -i test-container.tar
   assert_success
   assert_line --partial "Loaded image:"
 }
 
 # bats test_tags=containerize:piped-to-stdout
 @test "catalog: container is written to stdout when '-o -' is passed" {
+  skip "duplicate of next test"
```
Comment (mkenigs) [inline cli/tests/containerize.bats:127]: Can we just drop this test instead?

Comment (tomberek) [reply cli/tests/containerize.bats:127]: I suspect we will add it back in soon with variations in the upcoming sprints. No strong-opinion either way.

**Comment by mkenigs** [comment]
Test failure hopefully fixed by https://github.com/flox/flox/pull/2054, can you rebase?

**Comment by mkenigs** [comment]
> Test failure hopefully fixed by #2054, can you rebase?

Actually never mind it will get rebased in the merge queue

**Pull request merged**